pkg/network/aci_network_test.go

```diff
@@ -254,3 +257,83 @@ func TestShouldCreateSubnet(t *testing.T) {
 	}
 
 }
+
+func TestValidateNetworkConfig(t *testing.T) {
+	azConfig := auth.Config{}
+	err := azConfig.SetAuthConfig(context.TODO())
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	pn := &ProviderNetwork{}
```

Comment:
We need to move the pn inside the for [loop](https://github.com/virtual-kubelet/azure-aci/pull/537/files#diff-b6968d67da7e5445402e60e9f7865636274e1d7c85ad4c3b90276e4de34f27bfR323) so we are sure it's getting reset after each case.

Reply:
done